dbflute-engine/src/main/java/org/dbflute/properties/DfSimpleDtoProperties.java

```diff
@@ -156,7 +157,7 @@ protected String doGetOutputDirectory(String value) {
         final String baseDir = getBasicProperties().getGenerateOutputDirectory();
         if (Srl.is_NotNull_and_NotTrimmedEmpty(value)) {
             if (value.startsWith("~/")) {
-                return "./" + Srl.substringFirstRear(value, "~/");
+                return DfEngineWorkDir.toPath("./" + Srl.substringFirstRear(value, "~/"));
             } else {
                 return baseDir + "/" + value;
```

Comment:
Should a specified path be filtered?
(but it may be absolute path...)

```
# relative path (should be filtered...?)
generateOutputDirectory = ../src/main/java

# absolute path (should not be filtered?)
generateOutputDirectory = /Users/sea/works/docksidestage/workspace/maihama/src/main/java
```